File: app/src/main/java/parkhomov/andrew/lensthicknesscalculator/activities/tabs/AbstractTabFragment.java
package parkhomov.andrew.lensthicknesscalculator.activities.tabs;

import android.app.Activity;
import android.support.v4.app.Fragment;
import android.support.v4.app.FragmentActivity;

import java.util.List;


public abstract class AbstractTabFragment extends Fragment {

    protected String title;
    protected Activity activity;

    protected List<String> headers;
    protected List<String> description;
    protected List<Integer> images;

    public String getTitle() {
        return title;
    }
}

File: app/src/main/java/parkhomov/andrew/lensthicknesscalculator/tabs/AbstractTabFragment.kt
package parkhomov.andrew.lensthicknesscalculator.tabs

import android.support.v4.app.Fragment


abstract class AbstractTabFragment : Fragment() {

    var title: String? = null
        protected set

    var headers: MutableList<String>? = null
        protected set

    var description: MutableList<String>? = null
        protected set

    var images: MutableList<Int>? = null
        protected set

}
